### PR TITLE
Fixes #33204 - host info without Puppet

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    context 'without Puppet' do
+      let(:host) do
+        FactoryBot.build(:host,
+          location: taxonomies(:location1),
+          organization: taxonomies(:organization1),
+          operatingsystem: operatingsystems(:redhat))
+      end
+
+      it 'doesnt fail without puppet facet' do
```
Comment (timogoebel) [inline test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb:30]:
```suggestion
      it 'does not fail without puppet facet' do
```

**Review by timogoebel — COMMENTED** [review]
Looks good, just needs a rubocop fix.

**Review by timogoebel — APPROVED** [review]

File: test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      let(:host) do
+        FactoryBot.build(:host, :with_puppet_enc,
+          location: taxonomies(:location1),
+          organization: taxonomies(:organization1),
+          operatingsystem: operatingsystems(:redhat),
+          config_groups: [config_group])
+      end
+
+      it 'adds config_groups to host parameters' do
+        assert_equal(cg_info.host_info['parameters']['foreman_config_groups'], [config_group.name])
```
Comment (kamils-iRonin) [inline test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb:18]: expected value should be passed as first argument, if I understand correctly, `[config_group.name]` is the expected value and `cg_info.host_info['parameters']['foreman_config_groups']` is the actual value, right?

```suggestion
        assert_equal([config_group.name], cg_info.host_info['parameters']['foreman_config_groups'])
```

**Review by kamils-iRonin — APPROVED** [review]

File: test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb (hunk 2 above)
Comment (timogoebel) [reply test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb:18]: Ah, I checked that but apparently not good enough.

**Comment by ezr-ondrej** [comment]
CP 1.0: ee49ade

**Pull request merged**